models/dodo/bnb/dodo_aggregator_bnb_trades.sql

```diff
@@ -20,78 +21,79 @@ WITH dexs AS
         -- dodo proxy
         SELECT
             evt_block_time AS block_time,
-            'DODO' AS project,
+            'DODO X' AS project,
             '0' AS version,
             sender AS taker,
-            '' AS maker,
+            CAST(NULL AS VARBINARY) AS maker,
             fromAmount AS token_bought_amount_raw,
             returnAmount AS token_sold_amount_raw,
             cast(NULL as double) AS amount_usd,
             fromToken AS token_bought_address,
             toToken AS token_sold_address,
             contract_address AS project_contract_address,
             evt_tx_hash AS tx_hash,
-            CAST(ARRAY() as array<bigint>) AS trace_address,
+            CAST(NULL as array<bigint>) AS trace_address,
             evt_index
         FROM
             {{ source('dodoex_bnb','DODOV2Proxy02_evt_OrderHistory')}}
         {% if is_incremental() %}
-        WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+        WHERE evt_block_time >= date_trunc('day', now() - interval '7' day)
         {% endif %}
 
         UNION ALL
 
         -- DODORouteProxy
         SELECT
             evt_block_time AS block_time,
-            'DODO' AS project,
+            'DODO X' AS project,
             '0' AS version,
             sender AS taker,
-            '' AS maker,
+            CAST(NULL AS VARBINARY) AS maker,
             fromAmount AS token_bought_amount_raw,
             returnAmount AS token_sold_amount_raw,
             cast(NULL as double) AS amount_usd,
             fromToken AS token_bought_address,
             toToken AS token_sold_address,
             contract_address AS project_contract_address,
             evt_tx_hash AS tx_hash,
-            CAST(ARRAY() as array<bigint>) AS trace_address,
+            CAST(NULL as array<bigint>) AS trace_address,
```

Comment:
Problem identified:
null comparison involved in the incremental updates.

Reply:
Just confirmed with @owen05, all `trace_address` in dodo_aggregator.trades are null.